evap/rewards/tests/test_views.py

```diff
@@ -84,7 +84,7 @@ class TestEventsView(WebTestWith200Check):
 
     @classmethod
     def setUpTestData(cls):
-        baker.make(UserProfile, email='manager@institution.example.com', groups=[Group.objects.get(name='Manager')])
+        make_manager()
```

Comment:
In this case, I wish that this user gets assigned somewhere – else I first think that it is unused and don’t know where it is actually needed. Similar to my prior comment,
```python
cls.test_users = [make_manager()]
```
(And of course in all other subclasses of `WebTestWith200Check` where this is sensible)